hardware/i2c-shared.md

```diff
@@ -17,7 +17,7 @@ The V1.5 micro:bit has a footprint for two different motion sensors: one made by
 
 ### I2C block diagram
 
-![I2C block](/docs/hardware/assets/i2c-block.svg)
+![I2C block](/docs/hardware/assets/i2c-diagram.png)
```

Comment:
Is the the page for micro:bit v1? If so we need to change this image, as i2c-diagram.png  is only applicable to v2.

Reply:
This file 'v1-block.svg' would do the trick? 
![image](https://user-images.githubusercontent.com/36864441/109135950-85739b80-774f-11eb-95b5-2d5caa367039.png)

Reply:
Where is this image used? I think it probably contains a lot more than the I2C stuff.

Reply:
the only reference i can see to it is in "1-3-revision.md".

Reply:
Right, makes sense to have a full diagram in that page. For the v1 I2c page we only really need the nrf51, motion sensors and edge connector part of that block diagram.

Reply:
![image](https://user-images.githubusercontent.com/36864441/109154848-c3c88500-7766-11eb-896d-64f4e46883aa.png)
How's this?

Reply:
That's great!
Is it simple to move the boxes around? I'm thinking maybe the image will look a bit too large as it is longer vertically than horizontally, if we could move the boxes around to be more "landscape" it would look better on the website.

I think we can also remove (LSM & FXOS), as those are specific to v1.5 (the sensors were different in v1.3x and had to be changed to these in v1.5), in that case we should also change it to say "Motion Sensors" in plural.

We can also remove the GPIO labels inside the nRF51 box.

Reply:
![image](https://user-images.githubusercontent.com/36864441/109201626-f9d32c80-7799-11eb-9565-f7ec155b829f.png)

Super simple, now.

Reply:
Great! let's change nrf51 to nRF51 and we'll be done, thanks Ross!